WcaOnRails/db/migrate/20160502113530_add_preferred_events_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddPreferredEventsToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :preferred_event_ids, :text
```

Comment:
It would be indeed if the Event would be an ActiveRecord, but since it's a 'virtual' model I think it's not really helpful. I forgot that Jeremy suggested to use serialized Array, what do you think about it? Are there still benefits from using the separate table which I'm missing?